skbio/sequence/_iupac_sequence.py

```diff
@@ -537,6 +539,38 @@ def expand_degenerates(self):
         return (self._to(sequence=''.join(nondegen_seq)) for nondegen_seq in
                 result)
 
+    @stable(as_of='0.4.0-dev')
+    def to_regex(self):
+        """Return a regex object that accounts for degenerate chars.
+
+        Returns
+        ------
+        regex
```

Comment:
Can you replace this with `regular expression object`? This will match how we've been documenting regular expressions, e.g. `Sequence.find_with_regex`.